### DISCUSSION TITLE
Turning off Bluetooth

### DISCUSSION THREAD

**Discussion by yanrui89** [bitcraze/discussions#532] [Q&A] [answered]
May I know how to deactivate all the Bluetooth (STM32 and AIDeck) on the crazyflie?

**Comment by krichardsson** [comment, marked as answer]
STM:
There is a mechanism that disables Bluetooth as soon as a packet from the Crazyradio is received, all you have to do is connect with a Crazyradio. 
If you want to prevent Bluetooth from being enabled at all, you have to [rebuild and flash the NRF firmware](https://www.bitcraze.io/documentation/repository/crazyflie2-nrf-firmware/master/build/build/) without BLE

``` make
make BLE=0
```

There is no Bluetooth on the AI-deck. It is maybe possible to get Bluetooth out of the ESP (I'm not sure) but it is not enabled in the firmware.

**Reply by yanrui89** [reply]
Thank you!
